blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -302,13 +300,13 @@ export const actions = {
      * @param config Application configuration.
      * @param organizationName (optional)
      */
+    // eslint-disable-next-line no-unused-vars
     fetchPipelinesIfNeeded(config, organizationName) {
         return (dispatch, getState) => {
             const pipelines = getState().adminStore.pipelines;
             const baseUrl = config.getAppURLBase();
-            const url = !organizationName ?
-                `${baseUrl}/rest/search/?q=type:pipeline` :
-                `${baseUrl}/rest/organizations/${organizationName}/pipelines/`;
+            // TODO: update this code to call /search with organizationName once JENKINS-36273 is ready
+            const url = `${baseUrl}/rest/search/?q=type:pipeline`;
```

Comment:
If we fetch the pipelines for an organization, right now it doesn't return any jobs that are nested in folders, which basically results in a null pipeline in the store. For now we'll use the full fetch for all pipelines, and eventually tweak this code back to add organizationName as a filter once JENKINS-32673 is complete.